class/segwit-p2sh-wallet.js

```diff
@@ -85,20 +85,21 @@ export class SegwitP2SHWallet extends LegacyWallet {
    * @return string Signed txhex ready for broadcast
    */
   createTx(utxos, amount, fee, address, memo, sequence) {
+    const newUtxos = JSON.parse(JSON.stringify(utxos));
```

Comment:
but why?

Reply:
few lines below utxos object is modified and because of that when we iterate through createTx we divide second time u.value by 100000000